payment_ippay_ach/views/payment_view.xml

```diff
@@ -0,0 +1,19 @@
+<odoo>
+
+    <record id='acquirer_form_ippay_ach' model='ir.ui.view'>
+        <field name='name'>acquirer.form.ippay.ach></field>
+        <field name='model'>payment.acquirer</field>
+        <field name='inherit_id' ref='payment.acquirer_form'/>
+        <field name='arch' type='xml'>
+            <xpath expr="//group[@name='acquirer']" position="after">
+                <group>
+                    <field name='api_url' widget="url" attrs="{'invisible': [('provider','!=', 'ippay_ach')],
+                    'required': [('provider','=', 'ippay_ach')]}"/>
+                    <field name='ippay_ach_terminal_id' attrs="{'invisible': [('provider','!=', 'ippay_ach')],
+                    'required': [('provider','=', 'ippay_ach')]}"/>
```

Comment:
This is also declared in python field declaration as required and we are repeating it in xml, please check this field declaration. 
Both fields: 1- api_url, 2- ippay_ach_terminal_id